client/account-recovery/controller.js

```diff
@@ -8,13 +8,19 @@ import page from 'page';
  * Internal dependencies
  */
 import LostPasswordPage from 'account-recovery/lost-password';
+import ForgotUsernamePage from 'account-recovery/forgot-username';
 import { getCurrentUser } from 'state/current-user/selectors';
 
 export function lostPassword( context, next ) {
 	context.primary = <LostPasswordPage basePath={ context.path } />;
 	next();
 }
 
+export function forgotUsername( context, next ) {
+	context.primary = <ForgotUsernamePage basePath={ context.path } />;
```

Comment:
I think the convention has been to use `controller.js` though I understand why technically it should be `.jsx`.